frontend-project/src/locales/pl-PL/menu.ts

```diff
@@ -6,4 +6,6 @@ export default {
   'menu.institutions': 'Instytucje',
   'menu.institutions.new': 'Nowa',
   'menu.letters': 'Listy',
+  'menu.letters.list': 'Lista',
```

Comment:
Lista listów (?)

Reply:
jest to dropdown menu, gdzie we wszystkich innych przypadkach używamy 'Lista'